pkg/operator/staticpod/controllers.go

```diff
@@ -108,6 +112,9 @@ type Builder interface {
 	WithMinReadyDuration(minReadyDuration time.Duration) Builder
 	WithStartupMonitor(enabledStartupMonitor func() (bool, error)) Builder
 
+	// Informs controllers to handle extra nodes as well as master nodes.
```

Comment:
I think that the convention is that the comment should begin with the name of the method.

Reply:
Yup, you're right, updating